touchforms/formplayer/api.py

```diff
@@ -321,35 +337,39 @@ def start_form_session(form_path, content=None, language="", session_data={}):
     """
     Start a new form session
     """
-    # TODO: this method has been deprecated and the config object 
+    # TODO: this method has been deprecated and the config object
     # should just be used directly. Temporarily left to support legacy code.
-    return XFormsConfig(form_path=form_path, 
+    return XFormsConfig(form_path=form_path,
                         instance_content=content,
                         session_data=session_data,
                         language=language).start_session()
-    
-def answer_question(session_id, answer, auth=None):
+
+
+def answer_question(session_id, answer, domain=None, auth=None):
     """
     Answer a question. 
     """
     data = {"action": "answer",
             "session-id": session_id,
-            "answer":answer }
-    return get_response(json.dumps(data), settings.XFORMS_PLAYER_URL, auth)
+            "answer": answer,
+            "domain": domain}
+    return get_response(json.dumps(data), auth)
 
-def current_question(session_id, auth=None):
+
+def current_question(session_id, domain=None, auth=None):
```

Comment:
@benrudolph no need to revert, i think we can just make a new pr that fixes the inconsistency